hello/test.py

```diff
@@ -0,0 +1,47 @@
+"""A program to practice using the LibCST visitor function."""
```

Comment:
This file can be removed. We are no longer using the visitor function, depreciating this file.